src/Aspire.Cli/Utils/CliUpdateNotifier.cs

```diff
@@ -31,7 +31,7 @@ public async Task NotifyIfUpdateAvailableAsync(DirectoryInfo workingDirectory, C
                 return;
             }
 
-            var availablePackages = await nuGetPackageCache.GetCliPackagesAsync(workingDirectory, prerelease: true, source: null, cancellationToken);
+            var availablePackages = await nuGetPackageCache.GetCliPackagesAsync(workingDirectory, true, null, cancellationToken);
```

Comment:
Nit: I sorta like the parameter names here, especially since source became nugetConfigFile.